ocaml/idl/sdk-README.html

```diff
@@ -1,110 +0,0 @@
-<html>
```

Comment:
I think that removing this file will break some targets in the ocaml/idl/OMakefile (e.g. noarch-install).